docker/Dockerfile

```diff
@@ -15,6 +15,7 @@ COPY ./service .
 # USER cleanapper
 
 EXPOSE 8080/tcp
-CMD [ "./service" ]
+# CMD [ "./service", "--mysql_password=${MYSQL_APP_PASSWORD}" ] # TODO: FIX!!!
```

Comment:
Let's remove a commented code and make a comment readable i.e. what is wrong with a current state and how to change it

Reply:
Right, done.